pkgs/applications/networking/protonmail-bridge/default.nix

```diff
@@ -6,7 +6,9 @@ let
 
   description = ''
     An application that runs on your computer in the background and seamlessly encrypts
-    and decrypts your mail as it enters and leaves your computer
+    and decrypts your mail as it enters and leaves your computer.
+
+    To work, gnome-keyring service must be enabled. See https://github.com/NixOS/nixpkgs/issues/45070 for details.
```

Comment:
Is it possible to briefly summarize the details instead of adding a link?

Reply:
I can just delete the link. As far as I understand, the gnome-keyring must be enabled and that's it...